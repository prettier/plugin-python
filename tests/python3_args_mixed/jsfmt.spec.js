run_spec(__dirname, ["python"], ">=3");
