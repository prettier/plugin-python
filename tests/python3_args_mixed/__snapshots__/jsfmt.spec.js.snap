// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`args_mixed.py 1`] = `
def hello(a, *, example=False, **kwargs):
    print("hello world", example)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
def hello(a, *, example=False, **kwargs):
    print("hello world", example)

`;
